root/usr/libexec/nethserver/lsm-wan-notify

```diff
@@ -34,8 +34,17 @@ fi
 
 DATE=$(date --date=@${TIMESTAMP})
 
-from=`/sbin/e-smith/config getprop firewall NotifyWanFrom`
-to=`/sbin/e-smith/config getprop firewall NotifyWanTo`
+
+toEmailRoot=`/sbin/e-smith/config getprop root EmailAddress`
+
+if [[ -n $toEmailRoot ]]; then
+    to=${toEmailRoot/,/ }
+    from=`/sbin/e-smith/config getprop root SenderAddress`
+else
+    from=`/sbin/e-smith/config getprop firewall NotifyWanFrom`
+    to=`/sbin/e-smith/config getprop firewall NotifyWanTo`
+fi
+
```

Comment:
Proposed solution will change the behavior of existing firewall where root mail is set.
Example:
- `EmailAddress` set to `test1@mydomain.org`
- `NotifyWanTo` set to `test2@mydomain.org`

Currently, notifications are sent to `test2@mydomain.org`, but after the patch, messages will be delivered to `test1@mydomain.org`.
We can't change current behavior.

Proposed patch: do not change the logic, just support multiple destinations since EmailAddress already supports it.
```suggestion
from=`/sbin/e-smith/config getprop firewall NotifyWanFrom`
to=`/sbin/e-smith/config getprop firewall NotifyWanTo || tr ',' ' '`
```